README.md

```diff
@@ -169,6 +169,55 @@ Alternatively, you can use an AUR helper:
 yay -S concord-git
 ```
 
+#### Lix/Nix/Nixos
```

Comment:
I might would write it as `NixOS/Nix/Lix` based on the prevalence of the choices listed